### ISSUE TITLE
Transform @Deprecated annotations into comments

### ISSUE THREAD

**Issue by SButterfly** [vojtechhabarta/typescript-generator#603]
Hi
I marked some properties in POJO classes as `@Deprecated(since = "don't use it")`, hoping frontend team will try not to use it.
But unfortunately this annotations are not passed to typescript.
Can they be passed smth like this:
```java
public class Person {
    public String fullName;
    @Deprecated(since = "Use fullname")
    public String name;
}
```

as

```ts
interface Person {
    fullName: string;
    /** @deprecated since: Use fullname */
    name: string;
}
```

**Comment by vojtechhabarta** [comment]
I implemented support for `@Deprecated` annotation used on classes, properties, enums, enum members and REST methods.

Element `since` is not supported because it was added in Java 9 but typescript-generator still supports Java 8. By the way, value of this element should be a version, not a reason.

**Comment by vojtechhabarta** [comment]
Released in [v2.28.785](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.28.785).
